src/mixins/embed.js

```diff
@@ -506,5 +515,25 @@ export default {
       }
       // TO-DO: handle go to portfolio
     },
+    async updateTempCTA() {
+      this.tempLink = `https://app.rinkeby.like.co/nfttest/fetch?url=${this.referrer}&liker_id=${this.id}`;
+      this.tempCTA = 'Create ISCN';
+      this.tempNFTPrice = -1;
+      if (this.iscnId) {
+        const res = await axios.get(`${LIKECOIN_API}/likernft/mint`, {
+          params: {
+            iscn_id: this.iscnId,
+          },
+        }).catch(() => { });
```

Comment:
write an api function in `src/util/api/api.js` instead of importing axios
also use `try catch` instead of `catch` when using `await`